src/server/StudioManagementSystem/Startup.cs

```diff
@@ -18,6 +23,27 @@ public Startup(IConfiguration configuration)
     /// <param name="services"></param>
     public void ConfigureServices(IServiceCollection services, IWebHostEnvironment env)
     {
+        // Add authentication.
+        services.AddAuthentication(options =>
+            {
+                options.DefaultAuthenticateScheme = JwtBearerDefaults.AuthenticationScheme;
+                options.DefaultChallengeScheme = JwtBearerDefaults.AuthenticationScheme;
+            })
+            .AddJwtBearer(options =>
+            {
+                options.TokenValidationParameters = new TokenValidationParameters
+                {
+                    ValidateIssuer = false,
+                    ValidateAudience = true,
+                    ValidateLifetime = true,
+                    ValidateIssuerSigningKey = true,
+                    ValidAudience = "authenticated",
+                    // to fix later
+                    IssuerSigningKey = new SymmetricSecurityKey(Encoding.UTF8.GetBytes("Pxje6J28POZQlq+nZNdNUzg7N/vBfwNb+U+dcthqKY1/5G6XqyEs1HZMSv6hhY8xhDQtHkc/+pt0cosGlgpfXQ==")),
```

Comment:
what's the issue here?

Reply:
We shouldn't be commitment this to the repo because than anyone can use the JWT master key committed to create their own token, but for the sake of simple development, I've just comitted it.

Reply:
^ aight gotcha, let's comment that. Didn't realise that was th e master key reading that quickly